NGitLab/Models/PipelineBasic.cs

```diff
@@ -33,4 +33,7 @@ public class PipelineBasic
 
     [JsonPropertyName("web_url")]
     public string WebUrl;
+
+    [JsonPropertyName("name")]
+    public string Name;
```

Comment:
Make this a property instead.

Granted, legacy code uses public fields, but we shouldn't add any more.